docs/compiler.md

```diff
@@ -31,6 +31,8 @@ a dialect of Go rather than a complete port of the language:
  * type assertion with two return values is not supported; single return value (of the desired type)
    is supported; type assertion panics if value can't be asserted to the desired type, therefore
    it's up to the programmer whether assert can be performed successfully.
+ * type aliases including the built-in `any` alias are supported.
+ * generics are not supported, but eventually will be, ref. https://github.com/nspcc-dev/neo-go/issues/2376.
```

Comment:
Not sure they will be supported properly, but partially maybe.

Reply:
Adjusted.

Reply:
Hope, they will. At least function arguments can be supported, it doesn't seem impossible.